<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadSettingWidget</class>
 <widget class="QWidget" name="DownloadSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选项设置</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>610</width>
     <height>450</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>4</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>选项设置</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>592</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>29</y>
     <width>461</width>
     <height>370</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#444444;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="first">
    <widget class="QLineEdit" name="downloadDirEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>48</y>
       <width>270</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="baseSetDownloadWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
font-size:13px;
font-weight:bold;</string>
     </property>
     <property name="text">
      <string>基本设置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="downloadDirButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>47</y>
       <width>60</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>更改目录</string>
     </property>
    </widget>
    <widget class="QLabel" name="downloadDirLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>下载目录</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="functionArea" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>125</width>
     <height>425</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,200)</string>
   </property>
   <widget class="QLabel" name="baseSetWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
font-size:14px;
color:rgb(80,80,80);
font-weight:bold;;</string>
    </property>
    <property name="text">
     <string>基本设置</string>
    </property>
   </widget>
   <widget class="DownloadFunctionTableWidget" name="normalFunTableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>45</y>
      <width>125</width>
      <height>125</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>410</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="functionAreaDisplay" native="true">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>29</y>
     <width>486</width>
     <height>425</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>functionAreaDisplay</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>functionArea</zorder>
  <zorder>confirmButton</zorder>
  <zorder>cancelButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DownloadFunctionTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">downloadsettingwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
